sympy/functions/special/tests/test_zeta_functions.py

```diff
@@ -134,6 +134,13 @@ def test_polylog_expansion():
     assert myexpand(polylog(-5, z), None)
 
 
+def test_polylog_values():
+    assert polylog(2, 2) == pi**2/4 - I*pi*log(2)
+    assert polylog(2, S.Half) == pi**2/12 - log(2)**2/2
+    for z in [S.Half, 2, (sqrt(5)-1)/2, -(sqrt(5)-1)/2, -(sqrt(5)+1)/2, (3-sqrt(5))/2]:
+        assert Abs(polylog(2, z) - polylog(2, z + 1e-13)).evalf() < 1e-12
```

Comment:
The numeric value computed by `mpmath` can be obtained by setting `evaluate=False`.
```
>>> polylog(2, 2)
pi**2/4 - I*pi*log(2)
>>> polylog(2, 2, evaluate=False)
polylog(2, 2)
>>> polylog(2, 2, evaluate=False).evalf()
2.46740110027234 - 2.1775860903036*I  # this is computed by mpmath polylog
```